lib/ui/notifications.dart

```diff
@@ -129,9 +179,14 @@ class NotificationsWidgetState extends AssistantState<NotificationsWidget> {
                             _keywordController.clear();
                           },
                           decoration: const InputDecoration(
+                              filled: true,
+                              fillColor: Colors.white54,
+                              border: OutlineInputBorder(),
+                              //contentPadding: EdgeInsets.all(8),
                               labelText: 'Keyword(s)',
-                              isDense: true,
-                              border: InputBorder.none),
+                              labelStyle: TextStyle(color:Color.fromRGBO(51, 51, 102, 1),
+                              fontSize: 18),
+                              isDense: true),
                         ),
                         width: 300,
```

Comment:
I changed it to
width: MediaQuery.of(context).size.width/2
instead of the fixed width to account for larger devices